lib/checkPullRequestJob.js

```diff
@@ -32,6 +32,163 @@ const JOB_REGEX = new RegExp(
   ].join('')
 );
 
+const CRON_REGISTRY_FILENAME = ['cron.yaml', 'main_cron.py', 'cron.py'];
+const CRON_JOB_REGEX = new RegExp(
+  [
+    '(?<addition>\\+)(?<classDefinition>class\\s)',
+    '(?<name>[a-zA-Z]{2,256})(?<suffix>Handler)(?<funDef>\\()'
+  ].join('')
+);
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem} file
+ */
+const getNewCronJobsFromFile = (file) => {
+  return getNewItemsFromFileByRegex(CRON_JOB_REGEX, file);
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem} file
+ */
+const addsNewCronJob = (file) => {
+  const newJobs = getNewCronJobsFromFile(file);
+  return newJobs.length > 0;
+};
+
+/**
+ * @param {string} job
+ * @param {import('probot').Octokit.PullsListFilesResponse} files
+ */
+const isCronRegistryUpdated = (job, files) => {
+  const registryFile = files.find(({ filename }) => {
+    return CRON_REGISTRY_FILENAME.includes(filename);
+  });
+
+  return registryFile && registryFile.patch.includes(job);
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} newJobFiles
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} changedFiles
+ */
+const getCronRegistryReminder = (newJobFiles, changedFiles) => {
+  let registryReminder = ', ';
+
+  const hasNotAddedFileToRegistry = newJobFiles.some(
+    (file) => {
+      const jobs = getNewJobsFromFile(file);
+      const jobModuleName = file.filename;
+      return jobs.some((job) => !isCronRegistryUpdated(
+        `${jobModuleName}.${job}`, changedFiles));
+    }
+  );
+
+  if (hasNotAddedFileToRegistry) {
+    const jobText = newJobFiles.length > 1 ? 'jobs' : 'job';
+    const jobRegistryFile = 'job registry'.link(
+      'https://github.com/oppia/oppia/blob/develop/cron.yaml');
+    registryReminder +=
+      'please add the new ' + jobText + ' to the ' + jobRegistryFile + ' and ';
+  }
+
+  return registryReminder;
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} newJobFiles
+ */
+const getCronJobNameString = (newJobFiles) => {
+  return getNameString(
+    newJobFiles,
+    {
+      singular: 'job',
+      plural: 'jobs'
+    },
+    CRON_JOB_REGEX
+  );
+};
```

Comment:
I think we do not need this. Check for registries is only relevant for normal jobs.

Reply:
Removed it!